rust/lib/src/lib.rs

```diff
@@ -17,6 +17,19 @@ lazy_static! {
     static ref LIGHTCLIENT: Mutex<RefCell<Option<Arc<LightClient>>>> =
         Mutex::new(RefCell::new(None));
 }
+use zingoconfig::ChainType;
+fn infer_chaintype(server_uri: &str) -> ChainType {
+    // Attempt to guess type from known URIs
+    match server_uri {
+        "https://mainnet.lightwalletd.com:9067/"
+        | "https://lwdv2.zecwallet.co:1443"
+        | "https://lwdv3.zecwallet.co:443" => ChainType::Mainnet,
+        "https://testnet.lightwalletd.com:9067" => ChainType::Testnet,
+        x if x.contains("127.0.0.1") | x.contains("localhost") => ChainType::Regtest,
```

Comment:
Just in case... you have to put the `/` slash at the end of all of the URI's... otherwise it will fail